props/saber_chris_buttons.h

```diff
@@ -0,0 +1,1036 @@
+//    FastOn();
+      On();
+        }
+        return true;
+#endif
+
+
+#ifdef SA22C_THRUST_ON
+      case EVENTID(BUTTON_NONE, EVENT_THRUST, MODE_OFF):
+        if (millis() - saber_off_time_ > 1000) {
+    FastOn();
+//      On();
+        }
+        return true;
+#endif
+
+
+#ifdef SA22C_FORCE_PUSH
+      case EVENTID(BUTTON_NONE, EVENT_PUSH, MODE_ON):
+        if (FORCE_PUSH_CONDITION &&
+            millis() - last_push_ > 2000) {
+          if (SFX_push) {
+            hybrid_font.PlayCommon(&SFX_push);
+          } else {
+            hybrid_font.DoEffect(EFFECT_FORCE, 0);
+          }
+          last_push_ = millis();
+        }
+        return true;
+#endif
+
+
+//  Multiple Skips only available with 2 button installs.
+//  Skips forward five fonts if pointing up, skips back five fonts if pointing down.
+case EVENTID(BUTTON_AUX, EVENT_SECOND_SAVED_CLICK_SHORT, MODE_OFF):
+    // backwards if pointing down
+    SetPreset(current_preset_.preset_num + (fusor.angle1() < -M_PI / 4 ? -5 : 5), true);
+#ifdef SAVE_PRESET
+    SaveState(current_preset_.preset_num);
+#endif
+    return true;
+
+ 
+// Skips forward ten fonts if pointing up, skips back ten fonts if pointing down.
+case EVENTID(BUTTON_AUX, EVENT_THIRD_SAVED_CLICK_SHORT, MODE_OFF):
+    // backwards if pointing down
+    SetPreset(current_preset_.preset_num + (fusor.angle1() < -M_PI / 4 ? -10 : 10), true);
+#ifdef SAVE_PRESET
+    SaveState(current_preset_.preset_num);
+#endif
+    return true;
+
+   
+// Saber ON AND Volume Adjust.
+case EVENTID(BUTTON_POWER, EVENT_FIRST_SAVED_CLICK_SHORT, MODE_OFF):
+    if (!mode_volume_) {
+      On();
+    } else {
+     if (fusor.angle1() > 0) {
+      VolumeUp();
+    } else {
+      VolumeDown();
+    }
+    }
+    return true;
+
+
+// Modified 2 and 3 button 'next/previous preset' AND volume down,
+// to align with multiple skips above.
+// Forward one font pointing up, back one font pointing down.
+  case EVENTID(BUTTON_AUX, EVENT_FIRST_SAVED_CLICK_SHORT, MODE_OFF):
+    // backwards if pointing down
+    if (!mode_volume_) {
+    SetPreset(current_preset_.preset_num + (fusor.angle1() < -M_PI / 4 ? -1 : 1), true);
+        } else {
+      VolumeDown();
+    }
+#ifdef SAVE_PRESET
+    SaveState(current_preset_.preset_num);
+#endif
+    return true;
+
+
+// 1 button 'next/previous preset' command.
+// Forward one font pointing up, back one font pointing down.
+#if NUM_BUTTONS == 1 
+      case EVENTID(BUTTON_POWER, EVENT_SECOND_HELD, MODE_OFF):
+    // backwards if pointing down
+    if (!mode_volume_) {
+    SetPreset(current_preset_.preset_num + (fusor.angle1() < -M_PI / 4 ? -1 : 1), true);
+        } else {
+      VolumeDown();
+    }
+#ifdef SAVE_PRESET
+    SaveState(current_preset_.preset_num);
+#endif
+    return true;
+#endif
+
+
+//  1 Button skips to first preset (up) or last preset (down)
+//  or middle preset if horizontal:
+#if NUM_BUTTONS == 2
+case EVENTID(BUTTON_AUX, EVENT_FIRST_HELD_LONG, MODE_OFF):
```

Comment:
this code is not indented correctly, please fix

Reply:
As above - hopefully now sorted.